plugins/scaffolder-actions/scaffolder-backend-argocd/src/actions/run/argocd.test.ts

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 import { PassThrough } from 'stream';
-import { getVoidLogger } from '@backstage/backend-common';
 import { TemplateAction } from '@backstage/plugin-scaffolder-node';
 import { ConfigReader } from '@backstage/config';
 import { createArgoCdResources } from './argocd';
+import { Logger } from 'winston';
```

Comment:
There's a new `mockServices` export for this from a new backend Backstage helper.

https://github.com/backstage/backstage/blob/master/docs/backend-system/building-plugins-and-modules/02-testing.md#testing-backend-plugins-and-modules

Reply:
Yep like this:

mockServices.logger.mock()

Reply:
It's made for unit testing. Any reason that you aren't using it here?